src/main/java/banking/dao/AccountDAO.java

```diff
@@ -14,9 +14,8 @@
     private static final String INSERT_INTO_ACCOUNT = "INSERT INTO account (number, pin) VALUES (?, ?)";
     private static final String SELECT_COUNT_NUMBER = "SELECT count(number) FROM account WHERE number = (?)";
     private static final String SELECT_ACCOUNT_WHERE_NUMBER_AND_PIN = "SELECT * FROM account WHERE number=? AND pin=?";
```

Comment:
You missed this one. It should be `SELECT_ACCOUNT_BY_NUMBER_AND_PIN `

Reply:
sorry, renamed now